packages/itwinui-css/backstop/tests/progress-indicator.html

```diff
@@ -1039,17 +1041,18 @@ <h3>Indeterminate</h3>
     <h2>Overlay</h2>
     <section id="demo-overlay">
 
-      <div id="demo-overlay-content">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
-        magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
-        consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
-        pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est
-        laborum.
-
+      <div class="demo-overlay-wrapper">
         <div
-          class="iui-progress-indicator-overlay"
-          tabindex="0"
+          class="demo-overlay-content"
+          inert
```

Comment:
We will be adding `inert` to content under overlay in react implementation?

Reply:
I'm not sure. I think we could ask the user to do it or offer an `<Inert>` utility component that imports the polyfill? Doesn't need to be decided right now.